apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -70,6 +110,11 @@
             {{/parse_error}}
         </span>
     </span>
+{{#multiline}}
+    {{#suffix}}
+    <label for="pl-string-input-input-{{uuid}}" class="form-label">{{{suffix}}}</label>
```

Comment:
Consider styling the suffix with [`.form-text`](https://getbootstrap.com/docs/5.3/forms/form-control/#form-text), which renders like this:

<img width="264" alt="Screenshot 2024-08-27 at 16 00 32" src="https://github.com/user-attachments/assets/e04b4466-bfcd-4670-aad9-2150132398cd">

Reply:
Would that go in this file? https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/elements/pl-string-input/pl-string-input.css

Reply:
No. Just use the `form-text` class instead of `form-label` on the line I commented on.

Reply:
Changed the suffix here: https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/7cf4747c7a558d3012f7c664b1bd483e6bb00915